pkg/recommendation/capacity.go

```diff
@@ -150,6 +150,7 @@ func (r *capacityRecommendationRequest) GetRecommendation() map[string]CapacityR
 			resultMap[kind] = cr
 		}
 	}
+
```

Comment:
It will be great to explain in PR message how a device recommendation looks like e.g. its request & response.

Reply:
@slalwani97 It is very important to have all the changes Unit Tested. Please add the UT